src/components/waitlist/WaitlistPanel.tsx

```
@@ -0,0 +1,204 @@
+          </div>
+          <button
+            type="button"
+            onClick={() => setShowDetails(!showDetails)}
+            className="neu-border p-2 hover:bg-gray-50 transition-colors"
+            aria-label={showDetails ? "Hide details" : "Show details"}
+          >
+            {showDetails ? (
+              <ChevronUp className="h-4 w-4" />
+            ) : (
+              <ChevronDown className="h-4 w-4" />
+            )}
```

ESLint

[error] Replace `(⏎··············<ChevronUp·className="h-4·w-4"·/>⏎············)·:·(⏎··············<ChevronDown·className="h-4·w-4"·/>⏎············)` with `<ChevronUp·className="h-4·w-4"·/>·:·<ChevronDown·className="h-4·w-4"·/>`

(prettier/prettier)